### PR TITLE
K8s video link

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,14 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
 
 # Watch the Video
 
+#### Configure and run Watson Conversation Slots Intro
```
Comment (stevemart) [inline README.md]: how about "Running this application with Cloud Foundry on Bluemix"

File: README.md [hunk 2]
```diff
@@ -29,8 +29,14 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
+
 [![](http://img.youtube.com/vi/6QlAnqSiWvo/0.jpg)](https://youtu.be/6QlAnqSiWvo)
 
+#### Run the application in a container on Bluemix with Kubernetes
```
Comment (stevemart) [inline README.md]: how about "Running this application in a container with Kubernetes on Bluemix"

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: video does show "Deploy to Bluemix", but also local deploy. Does that matter?

Comment (stevemart) [reply README.md]: in the end it runs as an application with CF on Bluemix, so i don't think it's misleading :)

**Pull request merged**